src/custom-element.js

```diff
@@ -418,6 +418,7 @@ export function createAmpElementProto(win, name, implementationClass) {
     if (this.isUpgraded()) {
       this.implementation_.layoutWidth_ = this.layoutWidth_;
     }
+    this.implementation_.onLayoutMeasure();
```

Comment:
Should only do it when upgraded. E.g. in the `if` above. But we will also need to consider how to set it after upgrade.